ansible/roles/operator-pipeline/templates/openshift/tasks/cert-project-check.yml

```diff
@@ -30,4 +41,15 @@ spec:
           exit 1
         fi
 
+        PROJECT_URL="$(params.pyxis_url)/v1/projects/certification/id/$CERT_PROJECT_ID"
```

Comment:
Pipeline parameter pyxis_url uses a trailing `/` so I am not really sure if this concatenation would work.

Reply:
Just tested on my command line, it should work with the double `/`. I thought adding the `/` is safer in case the input doesn't contain a trailing one.